### PR TITLE
Qwik/pr issue pagination fix

### PR DESCRIPTION
**Pull request by hdJerry**
# Description

Steps to reproduce:

- go to the repository's pull request or issues page
- try to navigate between pages with "next" and "prev" buttons at the bottom of the page
- everything works fine

- try to sort PR or Issues with the sort dropdown by some "sortOptions"
- try again to navigate between pages 
- seems that the first click on the next button works well, but If we try to click "prev" and then "next" button, something goes wrong and we see a white page with no PRs or Issues.

### CODE REVIEW

File: qwik-graphql-tailwind/src/components/pull-request-issue-tab/pull-request-issue-tab.tsx [hunk 1]
```diff
@@ -31,6 +32,9 @@ export const PullRequestIssueTab = component$(
   ({ openCount, closedCount, tabType, milestonesOption, labelOption, sortOption }: PullRequestIssueTabParams) => {
     const tab = tabType === 'issue' ? useContext(IssuesPRContext) : useContext(PullRequestContext);
     const dropdown = useContext(DropdownStores);
+    const { pathname } = useLocation();
+
+    // const basePath = `${params.owner}/${params.name}`;
```
Comment (Megio) [inline qwik-graphql-tailwind/src/components/pull-request-issue-tab/pull-request-issue-tab.tsx:37]: I think we can remove this comment

**Review by Megio — COMMENTED** [review]
Left a little comment, but overall LGTM 🚀

File: qwik-graphql-tailwind/src/components/pull-request-issue-tab/pull-request-issue-tab.tsx (hunk 1 above)
Comment (hdJerry) [reply qwik-graphql-tailwind/src/components/pull-request-issue-tab/pull-request-issue-tab.tsx]: Thanks

**Review by Megio — APPROVED** [review]
LGTM

**Pull request merged**